docs/versioned_docs/version-0.1.6/api/06-Configuration/askui-ui-controller-server.md

```diff
@@ -0,0 +1,130 @@
+---
+custom_edit_url: null
+---
+
+# Askui-UI-Controller-Server
+
+
+Before running your tests you have start the Askui-UI-Controll-Server.
+There are multiple arguments which can be modified und provided to configure the Askui-UI-Contoller-Server. 
+In our implementation we use an interface which define all properties which can be provided to the Ask-UI-Contoller-Server.
+In the following we present you a short explanation for all properties.
+We also have an example to setup an controll server with specific values at the end of this page.
+
+## Table of contents
+
+### Properties
+
+- [binaryVersion](#binaryversion)
+- [display](#display)
+- [host](#host)
+- [logFilePath](#logfilepath)
+- [logLevel](#loglevel)
+- [minimize](#minimize)
+- [overWriteBinary](#overwritebinary)
+- [port](#port)
+
+## Properties
+
+### binaryVersion
+
+• **binaryVersion**: `string`
+
+There are different versions of The Askui-UI-Controller-Server.
```

Comment:
where can I find them?
  we need a table with different versions and the supported os and arch